client/src/setupTests.js

```diff
@@ -0,0 +1,7 @@
+import Enzyme, { configure, shallow, mount, render } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
+
+export { shallow, mount, render };
```

Comment:
May I ask is there a specific reason you export Enzyme, shallow, mount, and render here instead of importing directly from MortgageSection.test.js file?